package.json

```diff
@@ -7,9 +7,9 @@
     "node": "^6.0.0"
   },
   "scripts": {
-    "start": "bin/server",
-    "deploy": "bin/deploy",
-    "build": "node_modules/.bin/webpack"
+    "start": "node_modules/.bin/webpack-dev-server --inline --hot",
```

Comment:
Here we removed this `bin/server`. Does it means we can also remove the file "bin/server"?